metron-stellar/stellar-common/src/main/java/org/apache/metron/stellar/dsl/StellarFunction.java

```diff
@@ -23,4 +23,5 @@
   Object apply(List<Object> args, Context context) throws ParseException;
   void initialize(Context context);
   boolean isInitialized();
```

Comment:
if you are going to have a no parameter function, why not just implement closable?